resources/definitions/tenlog_printer.def.json

```diff
@@ -0,0 +1,293 @@
+{
+    "name": "tenlog_printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": { 
+        "visible": false,
+        "author": "Tenlog",
+        "manufacturer": "Tenlog",
+        "weight": 4,
+
+        "has_variants": false,
+
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "first_start_actions": ["MachineSettingsAction"],
+
+        "has_machine_quality": false,
+        "preferred_quality_type": "draft",
+        "preferred_material": "tlgreen_pla",
+
+        "machine_extruder_trains": {
+            "0": "tl_dual_0",
+            "1": "tl_dual_1"
+        },
+
+        "exclude_materials": [
```

Comment:
Since you're using a material diameter of 1.75mm, most of these materials aren't allowed in any case. Could you remove the listing for 2.85mm materials? (Those are the generics, the Ultimaker materials and the structur3d one.)